org.eclipse.scanning.event/src/org/eclipse/scanning/event/Constants.java

```diff
@@ -11,6 +11,8 @@
  *******************************************************************************/
 package org.eclipse.scanning.event;
 
+import java.util.concurrent.TimeUnit;
+
 public class Constants {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)